kalite/store/models.py

```diff
@@ -51,7 +51,7 @@ class StoreTransactionLog(DeferredCountSyncedModel):
     Model to track student 'expenditure' of points.
     """
 
-    # TODO-BLOCKER(rtibbles): Update this to "0.13.0" (or whatever the release version number is at the time this goes upstream)
+    # TODO(rtibbles): Update this to "0.13.0" (or whatever the release version number is at the time this goes upstream)
```

Comment:
Sure. But the store app isn't activated in 0.13.0, right? Not even clear to me where the minversion is used, is this just a duplicate of information that could be found in `version.py`?